src/status_im2/subs/communities.cljs

```diff
@@ -214,3 +214,30 @@
         (map #(assoc (get % 1) :community-id id))
         (sort-by :position)
         (into []))))
+
+(re-frame/reg-sub
+ :communities/categorized-channels
+ (fn [[_ community-id]]
+   [(re-frame/subscribe [:communities/community community-id])])
+ (fn [[{:keys [joined categories chats]}] [_ _]]
+   (reduce
+    (fn [acc [_ {:keys [name categoryID id emoji can-post?]}]]
+      (let [category (keyword (get-in categories [categoryID :name] "None"))]
```

Comment:
The `None` word should probably be translated.

Reply:
yeah and its better to use categoryID as a key not a keyword("Name")